test/cpp/aot_inductor/test.cpp

```diff
@@ -78,9 +79,9 @@ TEST(AotInductorTest, BasicTest) {
 
   AOTI_RUNTIME_ERROR_CODE_CHECK(AOTInductorModelContainerRun(
       container_handle,
-      inputs_handle,
+      input_handles.data(),
```

Comment:
steals!

Reply:
It's so wonky though, cuz the vector itself doesn't get stolen, only the elements inside it